src/com/sap/piper/tools/neo/NeoCommandHelper.groovy

```diff
@@ -87,6 +90,15 @@ class NeoCommandHelper {
         return "--source ${BashUtils.quoteAndEscape(source)}"
     }
 
+    private String extensions() {
```

Comment:
But I tried to mimic the style we have in that class.

Reply:
I also think that the name is not very good

Reply:
I would suggest to use getExtension(), so getVariableName().

I think that this should have been addressed before, when the other methods were implemented, but we did miss it in previous reviews.

Since it is not in the scope of this PR, it could be done in another PR afterwards.